garrysmod/lua/vgui/prop_combo.lua

```diff
@@ -10,13 +10,15 @@ function PANEL:Setup( vars )
 
 	self:Clear()
 
+	local bico = istable( vars.icons )
```

Comment:
This should be moved to just above the loop.

```
	local bico = istable( vars.icons )
	for id, thing in pairs( vars.values or {} ) do
```

should also probably have a better name like `hasIcons`.